maskrcnn_benchmark/engine/trainer.py

```diff
@@ -107,6 +108,37 @@ def do_train(
             )
         if iteration % checkpoint_period == 0:
             checkpointer.save("model_{:07d}".format(iteration), **arguments)
+            if data_loader_val is not None:
+                meters_val = MetricLogger(delimiter="  ")
+                synchronize()
+                with torch.no_grad():
+                    for idx_val, (images_val, targets_val, _) in enumerate(data_loader_val):
+                        images_val = images_val.to(device)
+                        targets_val = [target.to(device) for target in targets_val]
+                        loss_dict = model(images_val, targets_val)
+                        losses = sum(loss for loss in loss_dict.values())
+                        loss_dict_reduced = reduce_loss_dict(loss_dict)
+                        losses_reduced = sum(loss for loss in loss_dict_reduced.values())
+                        meters_val.update(loss=losses_reduced, **loss_dict_reduced)
+                synchronize()
+                logger.info(
+                    meters_val.delimiter.join(
+                        [
+                            "[Validation]: ",
+                            "eta: {eta}",
+                            "iter: {iter}",
+                            "{meters}",
+                            "lr: {lr:.6f}",
+                            "max mem: {memory:.0f}",
+                        ]
+                    ).format(
+                        eta=eta_string,
+                        iter=iteration,
+                        meters=str(meters),
```

Comment:
meters needs to be meters_val here, otherwise the training metrics are displayed

Reply:
Oh... Yes. Fixed.